cms/core/tests/test_context_processors.py

```diff
@@ -1,33 +1,41 @@
-import pytest
+from django.test import RequestFactory, TestCase
 from wagtail.models import Site
 
 from cms.core.context_processors import global_vars
 from cms.core.models import Tracking
 
-pytestmark = pytest.mark.django_db
 
+class ContextProcessorTestCase(TestCase):
+    """Tests for context processors."""
+    def setUp(self):
+        request_factory = RequestFactory()
```

Comment:
nit: why not set this up in `setUpTestData`?
Granted we have a couple of tests, but my preference for anything that can be global to go to `setUpTestData`

Reply:
Whilst the factory isn't really mutable, the request definitely could be (eg site caches), so i think it makes sense to do it per test. Separating the factory is possibly, but probably more noise than it's worth.